makefile.cargo

```diff
@@ -62,6 +71,9 @@ endif
 ifeq ($(OSTYPE),android)
 endif
 
+ifeq ($(OSTYPE),windows)
+endif
```

Comment:
Looks like it's a block to be filled in later if necessary. We have that for the other OSs too.